evennia/contrib/actions/README.md

```diff
@@ -0,0 +1,372 @@
+
+To have characters set their movement type during the game, such as through a "pace" command, ensure that the command does the following:
+
+>     <character object>.actions.movetype = <your string here>
+
+It is not necessary to include either movespeed or movetype. When you fail to include movespeed or movetype, the movement speed of the character will default to 1 distance unit per second. If, furthermore, the exit that the character chooses to traverse has a distance value of 1, then it will take 1 second to traverse it. Thus, you usually want to set the exits' distance values or give the characters movespeed functions, or both.
+
+The movebps function also has a single argument, a string representing the movement type being performed. You will want this string to be one you would assign to self.actions.movetype. The movebps function returns a string of comma-separated bodypart names, the same as you would find in an action's "bodyparts" item. While it is not necessary to set this function, the consequence of not doing so is that all movement actions will use no bodyparts at all, meaning that characters could issue a vast number of concurrent movement actions, as well as start moving to an exit immediately upon entering a room, rather than as soon as their MoveIn action was completed.
+
+When you subclass from __ActionRoom__, you should do: 
+
+> at_object_creation(self)
+>     self.actions.view = <your function here>
+>    *[...]*
+>     super(ActionRoom, self).at_object_creation()
+
+The view function must have two arguments. The first of these is the object that is being viewed and the second is the character viewing it. It will return either a formatted string or the value False, the former when the object being viewed can be seen by the viewer, and the latter when the object being viewed is invisible to the viewer. The string can contain the name of the object being viewed, its description or anything else the coder desires. The view function is used automatically inside the ActionRoom's "actions.display" method and should never need to be caled directly.
+
+Finally, when you subclass from __ActionExit__, you should do:
+> at_object_creation(self)
+>     self.distance = <your float here>
+>    *[...]*
+>     super(ActionExit, self).at_object_creation()
+
+The distance value of an exit represents how far one must travel to move into the room from the exit, and to move from the room to the exit. It is used in measuring travel times, together with the character's movement speed.
+
+The action system comes with two sets of __[commands](https://github.com/evennia/evennia/wiki/Commands "commands")__, one for characters and the other for staff. To load them, simply add them to one of your [CmdSets](https://github.com/evennia/evennia/wiki/Command%20Sets "command sets"), such as the ones in your game's commands/default_cmdsets.py file:
```

Comment:
It might be a good idea to make a short section higher up in this document that gives a short listing of all the various components you need to add to the game to try this contrib out. Something like

## Installation in brief
- Import and inherit your Character, Exit and Room classes from contrib.actions.ActionCharacter, ActionExit and ActionRoom 
- Customize these Attributes as needed (see below for details): ...
- Import and add the following cmdsets to your default cmdset: ...
- ... and so on 

Experienced Evennia users might get quite far with just that list and can dig into the details later as needed.